src/constants/controllerPath.ts

```diff
@@ -0,0 +1,12 @@
+export const controller_path = {
```

Comment:
@gerimujo496 Please keep a consistent naming convention for files, use either:
- Camel-case
- Snake-case
- Underscore
Not multiple of them at the same time